active/progress/ProgressRing2.md

```diff
@@ -0,0 +1,146 @@
+# Background
+>See proposal documents for:
+>
+> [determinate mode for ProgressRing](https://github.com/microsoft/microsoft-ui-xaml/issues/688) 
+>
+> [ProgressRing style](https://github.com/microsoft/microsoft-ui-xaml/issues/837)
+>
+> [Guidance for Progress controls](https://github.com/microsoft/microsoft-ui-xaml/issues/880)
+
+Progress controls indicate to a user that an operation is occuring. 
+Xaml has two controls for this:
+* the [ProgressBar](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.ProgressBar)
+control, which shows progress on a rectangle
+* and the
+[ProgressRing](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.ProgressRing)
+control, which shows progress on a circle.
+
+Progress _Bar_ has both a determinate and indeterminate mode.
+In determine mode the rectangle goes from empty to full, in indeterminate mode it shows an animation.
+Progress _Ring_ currently only has an indeterminate mode, but this spec is adding an indeterminate mode.
+
+ProgressBar is also getting a feature to replaces its built-in visualization (a spinning ring)
+with a custom animation defined as an 
+[IAnimatedVisualSource](https://docs.microsoft.com/uwp/api/Microsoft.UI.Xaml.Controls.IAnimatedVisualSource),
+which is known as a "Lottie Animation", and already supported (playable by) an
+[AnimatedVisualPlayer](https://docs.microsoft.com/uwp/api/Microsoft.UI.Xaml.Controls.AnimatedVisualPlayer).
```

Comment:
>>When Value changes it animates over some default duration. The animation has a natural duration and plays at its natural speed
>
>Recommend: explain the details. Should we make it configurable (like AVP's PlaybackRate)?

There should probably be a set short duration, with an easing as it reaches the value.  If it was a constant playback rate it could be frustrating waiting for the animation/transition to conclude.  The wider the gap between current value and new value, the faster the animation would be.

>>Reducing Value makes it loop forward and around back to the new value.
ProgressBar jumps backward but animates forward, ProgressRing animates forward
>
>Recommend: update to animate backwards?

It would be useful to expose the duration for the animated transitions as a StaticResource, so the developer could choose to set it to 0 and have instant changes.